doc/user/content/ingest-data/cdc-sql-server.md

```diff
@@ -0,0 +1,172 @@
+            "tasks.max": "1",
+            "topic.prefix": "server1",
+            "database.hostname": "sqlserver",
+            "database.port": "1433",
+            "database.user": "sa",
+            "database.password": "Password!",
+            "database.names": "testDB",
+            "schema.history.internal.kafka.bootstrap.servers": "<broker>:9092",
+            "schema.history.internal.kafka.topic": "schema-changes.inventory",
+            "key.converter": "io.confluent.connect.avro.AvroConverter",
+            "value.converter": "io.confluent.connect.avro.AvroConverter",
+            "key.converter.schema.registry.url": "http://<schema-registry>:8081",
+            "value.converter.schema.registry.url": "http://<schema-registry>:8081",
+            "database.encrypt": "false"
+        }
+    }
+    ```
+
+    You can read more about each configuration property in the [Debezium documentation](https://debezium.io/documentation/reference/2.4/connectors/sqlserver.html).
+
+{{< /tab >}}
+{{< /tabs >}}
+
+2.  Start the Debezium SQL Server connector using the configuration file:
+
+    ```sh
+    export CURRENT_HOST='<your-host>'
+    curl -i -X POST -H "Accept:application/json" -H "Content-Type:application/json" \
+    http://$CURRENT_HOST:8083/connectors/ -d @register-sqlserver.json
+    ```
+
+3.  Check that the connector is running:
+
+    ```
+    curl http://$CURRENT_HOST:8083/connectors/inventory-connector/status
+    ```
+
+Now, Debezium will capture changes from the SQL Server database and publish them to Kafka.
+
+### Create a Source
+
+Debezium emits change events using an envelope that contains detailed information about upstream database operations, like the `before` and `after` values for each record. To create a source that interprets the [Debezium envelope](/sql/create-source/kafka/#using-debezium) in Materialize:
+
+```sql
+CREATE SOURCE kafka_repl
+    FROM KAFKA CONNECTION kafka_connection (TOPIC 'server1.testDB.tableName')
+    FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY CONNECTION csr_connection
+    ENVELOPE DEBEZIUM;
+```
+
+#### Transaction Support
+
+Debezium provides [transaction metadata](https://debezium.io/documentation/reference/connectors/sqlserver.html#sqlserver-transaction-metadata) that can be used to preserve transactional boundaries downstream. Work is in progress to utilize this topic to support transaction-aware processing in Materialize ([#7537](https://github.com/MaterializeInc/materialize/issues/7537))!
+
+### Create a Materialized View
+
+Any materialized view defined on top of this source will be incrementally updated as new change events stream in through Kafka, resulting from `INSERT`, `UPDATE`, and `DELETE` operations in the original SQL Server database.
+
+```sql
+CREATE MATERIALIZED VIEW cnt_table AS
+    SELECT field1,
+           COUNT(*) AS cnt
+    FROM kafka_repl
+    GROUP BY field1;
+```
+
+## Known Limitations
```

Comment:
Also Microsoft has a [Known issues and errors with CDC](https://learn.microsoft.com/en-us/sql/relational-databases/track-changes/known-issues-and-errors-change-data-capture?view=sql-server-ver16)

Reply:
Good finding! I've added a link to this in the list